agent/app/agent.go

```diff
@@ -334,9 +338,25 @@ func (agent *ecsAgent) doStart(containerChangeEventStream *eventstream.EventStre
 		StringSetValue: aws.StringSlice(gpuIDs),
 	}
 
+	daemonDefinitions, err := md.ImportAll()
+	seelog.Errorf("Daemon import mountpoint failure: %s", err)
```

Comment:
todo Import all could return an array and error, or nil and error -- need to handle both cases -- error for single daemon failure or all daemon failure.
todo add note that we will not panic when daemons fail to import -- agent should continue running.  container instance health will act as the notification for this case